dotcom-rendering/src/web/lib/cardWrappers.tsx

```diff
@@ -109,6 +109,53 @@ export const Card100Media100 = ({
 	);
 };
 
+/**
+ * ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━┱┈┈┈┈┈┈┈┈┈┐
+ * ┃         ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒┃   25%   ┊
+ * ┃         ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒┃Remaining┊
+ * ┃         ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒┃         ┊
+ * ┗━━━━━━━━━━━━━━━━━━━━━━━━━━━┹┈┈┈┈┈┈┈┈┈┘
+ * Card designed to take up 75% of the container, with media that takes up 66%
+ *
+ * Options:
+ *  - Large headline
+ *  - Large image on the right (top on mobile)
+ *  - Trail text when there is no supporting content
+ *  - Up to 3 supporting content items, 1-2 aligned vertical, 3 aligned horizontal
+ */
+export const Card75Media66 = ({
+	trail,
+	showAge,
+	containerPalette,
+}: TrailProps) => {
+	return (
+		<FrontCard
+			trail={trail}
+			containerPalette={containerPalette}
+			showAge={showAge}
+			headlineSize="large"
+			headlineSizeOnMobile="large"
+			imageUrl={trail.image}
+			imageSize="large"
+			imagePosition="right"
+			imagePositionOnMobile="top"
+			trailText={
+				// Only show trail text if there is no supportContent
```

Comment:
Is this what we want? By looking at the frontend screenshots I can still see `trailText` even when there is `supportingContent`